scripts/elasticsearch-test-helpers/destination_connector/es_cluster_config.py

```diff
@@ -0,0 +1,11 @@
+import json
+import os
+
+CLUSTER_URL = "http://localhost:9200"
+INDEX_NAME = "ingest-test-destination"
+USER = os.environ["ELASTICSEARCH_USER"]
+PASSWORD = os.environ["ELASTICSEARCH_PASSWORD"]
+MAPPING_PATH = "docs/source/ingest/destination_connectors/elasticsearch_elements_mappings.json"
```

Comment:
was `docs/source/ingest/destination_connectors/data/elasticsearch_elements_mappings.json` for me

Reply:
If you mean if it's for testing yes it should help while testing this connector :) should help with manual testing, and is essential for CI

I also think it'll help users to setup their unstructured es indexes so we seem to be doing it for all destination connectors that require schemas

Reply:
I should have explained more. This was failing due to bad path.
```
Traceback (most recent call last):
  File "/Users/davidpotter/Documents/Unstructured/sessions/unstructured/scripts/elasticsearch-test-helpers/destination_connector/create_index.py", line 4, in <module>
    from es_cluster_config import (
  File "/Users/davidpotter/Documents/Unstructured/sessions/unstructured/scripts/elasticsearch-test-helpers/destination_connector/es_cluster_config.py", line 10, in <module>
    with open(MAPPING_PATH) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'docs/source/ingest/destination_connectors/elasticsearch_elements_mappings.json'
Stopping Elasticsearch Docker container
```

Reply:
Agreed, should be fixed now please try again